src/test/java/domain/lotto/ticket/LottoTest.java

```diff
@@ -0,0 +1,67 @@
+package domain.lotto.ticket;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class LottoTest {
+	@Test
+	@DisplayName("생성 테스트")
+	void valueOf() {
+		assertThat(Lotto.valueOf(Arrays.asList(1, 2, 3, 4, 5, 6))).isNotNull();
+	}
+
+	@Test
+	@DisplayName("생성 테스트 - 중복된 숫자가 있을 경우")
```

Comment:
```suggestion
	@DisplayName("생성 테스트 - 중복된 숫자가 있을 경우 예외 발생")
```
`DisplayName`에 발생하는 결과에 관한 내용이 포함된다면 테스트 코드를 직접 보지 않고도 어떤 결과를 일으키는지 알 수 있을 것 같아요 :)